tests/apollo/util/bft.py

```diff
@@ -53,8 +53,8 @@ def interesting_configs(selected=None):
         selected=lambda *config: True
 
     bft_configs = [#{'n': 4, 'f': 1, 'c': 0, 'num_clients': 4},
```

Comment:
Let's re-enable n=4, given that we have gained several minutes of execution time.

PS: This could be done in a separate PR for the sake of clarity.